test/e2e/framework/volume_util.go

```diff
@@ -537,6 +537,8 @@ func InjectHtml(client clientset.Interface, config VolumeTestConfig, volume v1.V
 
 	defer func() {
 		podClient.Delete(podName, nil)
+		err := waitForPodNotFoundInNamespace(client, injectPod.Name, injectPod.Namespace, PodDeleteTimeout)
```

Comment:
nit: s/injectPod.Name/podName/   I know injectPod.Name is the same as podName, but it is easy to understand the pod which is waited for its termination is the same as the deleted pod at line 539.

Reply:
fixed